### ISSUE TITLE
requirement for Spark 3.0.0 is inaccurate

### ISSUE THREAD

**Issue by alexnastetsky** [delta-io/delta#465]
The README says "Delta Lake currently requires Apache Spark 3.0.0", which is not true.
Only version 0.7.0 requires it.

**Comment by tdas** [comment]
Yes. making a PR to clarify that. the release section in the docs is more clear - https://docs.delta.io/latest/releases.html#compatibility-with-apache-spark

**Comment by mrdotmoon** [comment]
@tdas , one question for Delta Lake 0.7.0 ADLS gen2 supports.  Our production environment using Spark 3.0 and Hadoop 2.9.2, and we don't have plan to upgrade to Hadoop 3.2. Any issue if I submit job with Hadoop 3.2 jars? I tried looks like it works, but would like to understand any blockers here.

**Comment by zsxwing** [comment]
It’s likely you would hit some weird issues because there would be two versions of Hadoop jars in the classpath.
